bin/dune

```diff
@@ -0,0 +1,12 @@
+(executable
+ (name duniverse)
+ (public_name duniverse)
+ (libraries
+    cmdliner
+    duniverse_lib
```

Comment:
I'd prefer to call this library just `duniverse`, to follow the other ones.  The CLI driver can just be `main.ml` or `cli.ml` and then installed as `duniverse` via `(public_name duniverse)` in the executable stanza

Reply:
Wouldn't it make things more confusing in the end?

Happy to change it anyway but I tend to avoid giving the same name to two different things so that new comers to the project don't have to go through the dune files to figure out what is compiled to what.

Reply:
It's more confusing to not be able to predict the findlib name from the opam package name, I find. Either way, it doesn't matter much so I'll merge this as-is and we can change later if we want.  The `duniverse_lib` probably shouldn't be installed in the default package since it's really an internal library for the driver -- there are no expected external consumers outside of the CLI tool to start with are there?

Reply:
Not that I can think of right now.

It is indeed not installed atm!